src/serviceWorker/utils.test.ts

```diff
@@ -0,0 +1,67 @@
+import { PrecacheEntry } from 'workbox-precaching/_types'
+
+import { groupEntries } from './utils'
+
+describe('groupEntries', () => {
+  test('splits resources into onDemandCacheEntries and precacheEntries', () => {
+    const resources = [
+      './static/whitepaper.pdf',
```

Comment:
Is this a realistic resource entry?

Reply:
think so yes. it's just an example of a string url without a revision.